pkg/operations/kubernetesupgrade/upgradeagentnode.go

```diff
@@ -19,6 +18,7 @@ var _ UpgradeNode = &UpgradeAgentNode{}
 // UpgradeAgentNode upgrades a Kubernetes 1.5 agent node to 1.6
 type UpgradeAgentNode struct {
 	Translator              *i18n.Translator
+	TraceLogger             *logrus.Entry
```

Comment:
prefer name Logger or just logger. I don't think we need to export this.

Reply:
ok, thanks!

Reply:
we do need to export it :)